inc/define.php

```diff
@@ -649,4 +649,5 @@
     SoftwareLicense::getType()    => "pics/impact/softwarelicense.png",
     Supplier::getType()           => "pics/impact/supplier.png",
     User::getType()               => "pics/impact/user.png",
+    Database::getType()           => "pics/impact/databaseinstance.png",
```

Comment:
Maybe png files should be renamed to match their corresponding itemtype.
- `databaseinstance.png` -> `database.png`
- `database.png` -> `databaseinstance.png`

Reply:
done